### ISSUE TITLE
PV selector does not display correctly

### ISSUE THREAD

**Issue by Tom-Willemsen** [ISISComputingGroup/IBEX#4643]
On DEMO and SCIDEMO, opening the PV selector window causes a visual glitch. On dev machines, it appears to render correctly.

![image](https://user-images.githubusercontent.com/24375813/62937360-4d527500-bdc4-11e9-85d3-4f397e281440.png)

**Comment by Tom-Willemsen** [comment]
Potentially a windows 7 / windows 10 difference

**Comment by Tom-Willemsen** [comment]
@John-Holt-Tessella and I did some investigation around this issue.

We found that:
- This issue was in last release already, and we didn't get significant complaints from scientists about it as far as I can see.
- The rendering can be sorted out by dragging an unrelated window (e.g. windows explorer or notepad) over the top of the ibex window and then minimising/moving away again.
- It's not clear how to fix this from the IBEX side. @John-Holt-Tessella did some investigation about putting various different code on/off the UI thread but none of this helped

Given that it was already in last release, I think we can de-prioritise this and this is no longer a critical issue for the release
